docs/docs_server_configuration_reference.md

```diff
@@ -305,3 +305,21 @@ how they should be spawned: in interior or exterior. By default all the npcs are
   // ...
 }
 ```
+
+## weaponStaminaModifiers
```

Comment:
actually, should be *multipliers*, not *modifiers*. 

Modifier is + (plus)
Multiplier is * (multiply)

Reply:
I guess it is still should be a modifier and the reason is that we damage actor value by certain units [here](https://github.com/skyrim-multiplayer/skymp/pull/1671/files#diff-4481088266a71f94bd45bb76535e5f69628d98e8f7fb395635d14e399e86c23fR280), so actually we are subtracting something from someone.

Reply:
ok